hardware/settings.py

```diff
@@ -0,0 +1,3 @@
+
+#How many minutes a hardware request lasts
+REQUEST_TIME = 15
```

Comment:
Move this to hackathon_variables pls

Reply:
Maybe we could also add a datetime HACKATHON_STARTS or similar. So when it's passed that we show the hardware tab. What do you think?